### PR TITLE
[WNMGDS-864] Render simplified page view of the doc site

### PR DESCRIPTION
**Pull request by nichia**
## Summary
Render a view of the doc site that can be used to embed into the DSM.
This simplified page view would not contain the following:
- the left nav
- the site header
- the site footer
- the page content header

To see this view, add query string `?view=page` to the URL parameter. 
The goal is to have a single URL we can use to pull into the DSM that has code examples and guidance.

### Changed
This PR is updated to accept 3 types of `view` parameters on the URL:
- `?view=code` for the Live demos & code examples i.e. `https://design.cms.gov/components/button/?view=code`
- `?view=guidance` for guidance i.e. `https://design.cms.gov/components/button/?view=guidance`
- `?view=page` for the simplified page view i.e. `https://design.cms.gov/components/button/?view=page`

## How to test
Demo doc site: http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-864/doc-site-url-query-view
- View demo doc site and test that it works as per normal
- Add each of the `view` parameters to end of the URL to test that:
   - `?view=code` renders the Live demos & code examples (i.e. `https://design.cms.gov/components/button/?view=code`)
   - `?view=guidance` renders the guidance section (i.e. `https://design.cms.gov/components/button/?view=guidance`)
   - `?view=page` renders the simplified page view (i.e. `https://design.cms.gov/components/button/?view=page`)

### CODE REVIEW

File: packages/design-system-docs/src/scripts/components/Docs.jsx [hunk 1]
```diff
@@ -34,7 +35,17 @@ class Docs extends React.PureComponent {
   render() {
     const { routes, page } = this.props;
     const { menuOpen } = this.state;
-    return (
+    const view = typeof window !== 'undefined' && queryString.parse(window.location.search).view;
+    /**
+     * Hide site header, site footer, side nav when URL parameters query string 'view' contains
+     * 'basic', 'guidance' or 'page' (ie. design-system.cms.gov/components/button/?view=basic)
+     */
+    const hideHeaders = view === 'basic' || view === 'guidance' || view === 'page';
```
[the remark this answers is not in the document]
Comment (line47) [reply packages/design-system-docs/src/scripts/components/Docs.jsx:43]: Yeah I like that update

File: packages/design-system-docs/src/scripts/components/Page.jsx [hunk 2]
```diff
@@ -107,6 +121,13 @@ Page.defaultProps = {
 Page.propTypes = {
   depth: PropTypes.number,
   sections: PropTypes.arrayOf(PropTypes.shape(PageBlock.propTypes)),
+  /**
+   * URL parameters query string 'view':
+   * - basic: Page content minus Guidance section (excludes nav, header, footer, title)
```
Comment (line47) [inline packages/design-system-docs/src/scripts/components/Page.jsx:126]: Can we rename `basic` to be `code` or `examples`?

File: packages/design-system-docs/src/scripts/components/Page.jsx [hunk 3]
```diff
@@ -107,6 +121,13 @@ Page.defaultProps = {
 Page.propTypes = {
   depth: PropTypes.number,
   sections: PropTypes.arrayOf(PropTypes.shape(PageBlock.propTypes)),
+  /**
+   * URL parameters query string 'view':
```
Comment (line47) [inline packages/design-system-docs/src/scripts/components/Page.jsx:125]:
```suggestion
   * URL parameters query string 'view':
  All views exclude the  left nav, site header, site footer and page title
```

File: packages/design-system-docs/src/scripts/components/Page.jsx (hunk 2 above)
Comment (line47) [inline packages/design-system-docs/src/scripts/components/Page.jsx:126]:
```suggestion
   * - basic: Page code examples only
```

File: packages/design-system-docs/src/scripts/components/Page.jsx [hunk 4]
```diff
@@ -107,6 +121,13 @@ Page.defaultProps = {
...
+   * - basic: Page content minus Guidance section (excludes nav, header, footer, title)
+   * - guidance: Guidance section only
+   * - page: Page content with Guidance section (excludes nav, header, footer, title)
```
Comment (line47) [inline packages/design-system-docs/src/scripts/components/Page.jsx:128]:
```suggestion
   * - page: Page content with Guidance section
```

**Review by line47 — APPROVED** [review]
A few minor text updates but otherwise looks great!

**Review by kimieann — APPROVED** [review]
LGTM

**Pull request merged**